opt4j-optimizers/src/main/java/org/opt4j/optimizers/ea/aeseh/AeSeHModule.java

```diff
@@ -0,0 +1,134 @@
+package org.opt4j.optimizers.ea.aeseh;
+
+import static org.opt4j.core.config.annotations.Citation.PublicationMonth.UNKNOWN;
+
+import org.opt4j.core.config.annotations.Citation;
+import org.opt4j.core.config.annotations.Info;
+import org.opt4j.core.optimizer.OptimizerModule;
+import org.opt4j.core.start.Constant;
+import org.opt4j.optimizers.ea.Coupler;
+import org.opt4j.optimizers.ea.Selector;
+
+/**
+ * Module to bind the AeSeH evolutionary algorithm as optimizer. 
+ * 
+ * @author Fedor Smirnov
+ */
+@Info("Multi-objective evolutionary algorithm where the survival selection and the creation of neighborhoods is based on epsilon-dominance. The selection of parents is done within the created neighborhoods.")
+@Citation(authors = "Hernán Aguirre, Akira Oyama, and Kiyoshi Tanaka", title = "Adaptive ε-sampling and ε-hood for evolutionary many-objective optimization.", journal = "Evolutionary Multi-Criterion Optimization (EMO)", pageFirst = 322, pageLast = 336, year = 2013, month = UNKNOWN)
+public class AeSeHModule extends OptimizerModule {
```

Comment:
I would suggest that AeSeH should be an EA submodule as it is mainly an EA with a specific selector and coupler. Please add:
```
@Parent(EvolutionaryAlgorithmModule.class)
```